include/freerdp/server/rdpdr.h

```diff
@@ -35,20 +35,22 @@
 typedef struct _rdpdr_server_context RdpdrServerContext;
 typedef struct _rdpdr_server_private RdpdrServerPrivate;
 
+#ifndef __MINGW32__
 struct _FILE_DIRECTORY_INFORMATION
 {
 	UINT32 NextEntryOffset;
 	UINT32 FileIndex;
-	UINT64 CreationTime;
-	UINT64 LastAccessTime;
-	UINT64 LastWriteTime;
-	UINT64 ChangeTime;
-	UINT64 EndOfFile;
-	UINT64 AllocationSize;
+	LARGE_INTEGER CreationTime;
```

Comment:
This change is problematic for stable branch as the API breaks with it.

Reply:
According to MS docs [FILE_DIRECTORY_INFORMATION structure][1], those members are of LARGE_INTEGER type.

[1]: https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/ntifs/ns-ntifs-_file_directory_information